README.md

```diff
@@ -1,14 +1,14 @@
 # Single node cluster (snc) scripts for OpenShift 4 
 
+**NOTE:** Please select respective branch to create the bundle for specific openshift release (ie. to create 4.15.x openshift bundle choose `release-4.15` branch)
+
 ## How to use?
-- Make sure the one time setup is satisfied. (https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md#one-time-setup)
 - Clone this repo `git clone https://github.com/code-ready/snc.git`
 - `cd <directory_to_cloned_repo>`
 - `./snc.sh`
 
 ## How to create disk image?
 - Once your `snc.sh` script run successfully.
-- You need to wait for around 30 mins till cluster settle.
```

Comment:
Once snc.sh ran successfully, the cluster is fully operational, no need to wait?

Reply:
yes, now we don't need to wait anymore.